integration_test/cases/repo.exs

```diff
@@ -324,6 +324,7 @@ defmodule Ecto.Integration.RepoTest do
   end
 
   @tag :unique_constraint
+  @tag :unique_constraint_conflict
```

Comment:
The tests below also trigger conflict on unique_constraint. So is there a particular reason why this test does not pass on MSSQL?